Android 10 | gif from keyboard  => NPE getPickerIntentForSharing

Stack trace

````
Caused by: java.lang.NullPointerException: 
 
  at com.kbeanie.multipicker.utils.IntentUtils.getPickerIntentForSharing (IntentUtils.java:38)
 
  at im.vector.riotx.features.attachments.AttachmentsHelper.handleShareIntent (AttachmentsHelper.kt:180)
 
  at im.vector.riotx.features.home.room.detail.RoomDetailFragment.sendUri (RoomDetailFragment.kt:660)
 
  at im.vector.riotx.features.home.room.detail.RoomDetailFragment.onRequestPermissionsResult (RoomDetailFragment.kt:1027)
 
  at androidx.fragment.app.FragmentActivity.onRequestPermissionsResult (FragmentActivity.java:768)
 
  at android.app.Activity.dispatchRequestPermissionsResult (Activity.java:7458)
 
  at android.app.Activity.dispatchActivityResult (Activity.java:7309)

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -8,7 +8,9 @@ Improvements 🙌:
  - Implementation of /join command
 
 Bugfix 🐛:
- -  Message transitions in encrypted rooms are jarring #518
+ - Message transitions in encrypted rooms are jarring #518
+ - Images that failed to send are waiting to be sent forever #1145
+ - Fix / Crashed when trying to send a gif from the Gboard #1136
 
 Translations 🗣:
  -

--- a/vector/src/main/java/im/vector/riotx/features/attachments/AttachmentsHelper.kt
+++ b/vector/src/main/java/im/vector/riotx/features/attachments/AttachmentsHelper.kt
@@ -177,17 +177,26 @@ class AttachmentsHelper private constructor(private val context: Context,
     fun handleShareIntent(intent: Intent): Boolean {
         val type = intent.resolveType(context) ?: return false
         if (type.startsWith("image")) {
-            imagePicker.submit(IntentUtils.getPickerIntentForSharing(intent))
+            imagePicker.submit(safeShareIntent(intent))
         } else if (type.startsWith("video")) {
-            videoPicker.submit(IntentUtils.getPickerIntentForSharing(intent))
+            videoPicker.submit(safeShareIntent(intent))
         } else if (type.startsWith("audio")) {
-            videoPicker.submit(IntentUtils.getPickerIntentForSharing(intent))
+            videoPicker.submit(safeShareIntent(intent))
         } else if (type.startsWith("application") || type.startsWith("file") || type.startsWith("*")) {
-            filePicker.submit(IntentUtils.getPickerIntentForSharing(intent))
+            filePicker.submit(safeShareIntent(intent))
         } else {
             return false
         }
         return true
+    }
+
+    private fun safeShareIntent(intent: Intent): Intent {
+        // Work around for getPickerIntentForSharing doing NPE in android 10
+        return try {
+            IntentUtils.getPickerIntentForSharing(intent)
+        } catch (failure: Throwable) {
+            intent
+        }
     }
 
     private fun getPickerManagerForRequestCode(requestCode: Int): PickerManager? {

